run-build-functions.sh

```diff
@@ -437,6 +444,33 @@ install_dependencies() {
     restore_home_cache ".composer" "composer dependencies"
     composer install
   fi
+
+  # Go version
+  restore_home_cache ".gimme_cache" "go cache"
+  if [ -f .go-version ]
+  then
+    local goVersion=$(cat .go-version)
+    if [ "$installGoVersion" != "$goVersion" ]
+    then
+	installGoVersion="$goVersion"
+    fi
+  fi
+
+  if [ "$GIMME_GO_VERSION" != "$installGoVersion" ]
+  then
+    echo "Installing Go version $goVersion"
+    eval "$(GIMME_ENV_PREFIX=$HOME/.gimme_cache/env GIMME_VERSION_PREFIX=$HOME/.gimme_cache/versions gimme $goVersion)"
```

Comment:
Anyway we can avoid `eval`? Also, we usually fail the build if we fail to install the language version requested.